### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by LordHelmchen666** [JoomCoder-com/JoomCCK#118] [General]
Do you have any plans how to handle translations?
I did many of the german translation for Cobalt 8. Is it possible to reuse them for JoomCCK? Would save us a lot of work.
https://explore.transifex.com/mightytec/cobalt-8/

**Comment by andiamine** [comment]
yes i will do, and we are going to use crowdin instead of transifex.

**Comment by LordHelmchen666** [comment]
Here is my first german translation. You can install it like any extension via Joomla Installer.  Admin and global site translated, no fields or modules for now.
@andiamine you can use it on your site or just link here.

[com_joomcck_de-DE_v.1.0.zip](https://github.com/JoomCoder-com/JoomCCK/files/14593093/com_joomcck_de-DE_v.1.0.zip)

**Reply by LordHelmchen666** [reply]
Now here is full GERMAN translation.
[com_joomcck_de-DE_v.1.1.zip](https://github.com/JoomCoder-com/JoomCCK/files/14617921/com_joomcck_de-DE_v.1.1.zip)

**Reply by andiamine** [reply]
Your translation was added to https://crowdin.com/project/joomcck

**Comment by andiamine** [comment]
thanks, we are going to make some big changes on translations files.
- unify all fields language files
- remove duplicate constants.

**Comment by andiamine** [comment]
Everyone can help us translate here:
https://crowdin.com/project/joomcck

**Comment by Giorgi625** [comment]
Can I send you our translations for Finnish language for Cobalt 8? it will be very time consuming to translate it again. So if you can merge old translation with you current files, then we can translate missing translations only.

**Reply by andiamine** [reply]
Yea you can

**Comment by Giorgi625** [comment]
here is what we have translated in Finnish on Cobalt 
[Finnish JoomCCK.zip](https://github.com/user-attachments/files/16324897/Finnish.JoomCCK.zip)

**Reply by andiamine** [reply]
Thanks added to Crowdin but is just 25%

**Comment by andiamine** [comment]
Anyone can help us download translations from here:
https://explore.transifex.com/mightytec/cobalt-8/
